src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -41,18 +42,19 @@ public void ChatTestsStaff_001_twoUserEnterChat() {
   @DontRun(env = {"sandbox"})
   @Test(groups = {"ChatTestsStaff_002", "ChatStaff"})
   public void ChatTestsStaff_002_verifySwitchingBetweenMainAndPrivateSections() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)